### DISCUSSION TITLE
Sparse not showing up in stop

### DISCUSSION THREAD

**Discussion by Dr-McFish** [owlbarn/owl#652] [Q&A]
Hello, I have installed this library per the tutorial, it seams to be working. However, the "Sparse" module doesn't show up, while "Dense" does show up. Am I doing something wrong?

I am added
```
#require "owl-top"
open Owl
```
to `.ocamlinit` as per tutorial

![Screenshot 2024-01-04 at 23 28 28](https://github.com/owlbarn/owl/assets/61105022/0b8c4a92-1137-4344-829e-d91a7ae38353)

**Comment by mseri** [comment]
Looks like it was removed in 1.1: https://github.com/owlbarn/owl/commit/2182094b187665490c67df5fff5d2d4e8cf9f435

**Reply by mseri** [reply]
See also https://github.com/owlbarn/owl/pull/633

**Reply by Dr-McFish** [reply]
:O

**Reply by Dr-McFish** [reply]
thanks for the help
